app/views/welcome/index.html.erb

```diff
@@ -4,6 +4,13 @@
 <div class="col-md-6 col-md-offset-3 text-center">
     <h3>Future Home Of the Fall Gambit Chess App!</h3>
     <br />
-    <%= render "available_games" %>
+    <% if @games.present? %>
+      <%= render partial: 'available_games' %>
+    <% else %>
+      <h2>
+        No games available...
+      </h2>
+      <%= link_to 'Create Game', new_game_path, class: 'btn btn-success btn-large' %>
+    <% end %>
```

Comment:
definitely agree with you, @amarkpark. You can create a card in Trello for this (or you can make a github issue if you want to play around with that). that way we can keep it separate from the move_to method work. (sorry if this message is a repeat, i couldn't find it from the last time i submitted it)